test/library/standard/IO/append/multipleWriters.notest

```diff
@@ -0,0 +1 @@
+this is racy code right now
```

Comment:
I think this might be better as a `.noexec` or a future so we can keep the code up to date with library changes, etc.

Reply:
I changed this to a future with a `.noexec`